internal/app/kube_helpers.go

```diff
@@ -134,6 +135,50 @@ spec:
 
 }
 
+func setQuotas(ns string, quotas *quotas) {
+	if quotas == nil {
+		return
+	}
+
+	definition := `
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: resource-quota
+spec:
+  hard:
+`
+
+	for _, customQuota := range quotas.CustomQuotas {
+		definition = definition + Indent(customQuota.Name+": '"+customQuota.Value+"'\n", strings.Repeat(" ", 4))
+	}
+
+	//Special formatting for custom quotas so manually write these and then set to nil for marshalling
+	quotas.CustomQuotas = nil
+
+	d, err := yaml.Marshal(&quotas)
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+
+	definition = definition + Indent(string(d), strings.Repeat(" ", 4))
+
+	if err := ioutil.WriteFile("temp-ResourceQuota.yaml", []byte(definition), 0666); err != nil {
+		log.Fatal(err.Error())
+	}
+
+	cmd := kubectl([]string{"apply", "-f", "temp-ResourceQuota.yaml", "-n", ns}, "Creating ResourceQuota in namespace [ "+ns+" ]")
+	result := cmd.exec()
+
+	if result.code != 0 {
+		log.Fatal("ERROR: failed to create ResourceQuota in namespace [ " + ns + " ]: " + result.errors)
+	}
+
+	deleteFile("temp-ResourceQuota.yaml")
```

Comment:
I'd rather put this line before we run log.Fatal for result.code != 0 so the file "always" gets deleted even when kubectl apply failed

Reply:
Also you might want to use this catalog https://github.com/Praqma/helmsman/blob/master/internal/app/main.go#L10 for such file